README.md

```diff
@@ -1,59 +1,61 @@
-# teracy-dev-core
+# Teracy-dev-core
 
-The core extension for teracy-dev which support all options from https://www.vagrantup.com/docs/
+The core extension for [teracy-dev](https://github.com/teracyhq/dev/tree/develop) which support all options from [vagrant](https://www.vagrantup.com/docs/)
 for teracy-dev yaml config format.
 
+## Prerequisites
+
+- Follow the guide at https://github.com/teracyhq/dev/blob/develop/docs/getting_started.rst to set up `teracy-dev`.
 
 ## How to use
 
 By default, teracy-dev already configures to use this extension, however, you can override the
 config with your own, for example, to use a different version or a forked repo.
 
-To override the built-in config, create `workspace/teracy-dev-entry/config_default.yaml` with the
+To override the built-in config, create `workspace/teracy-dev-entry/config_override.yaml` with the
```

Comment:
config_default.yaml

config_override.yaml should be used only by users on their own to override the existing config.